src/services/dex.js

```diff
@@ -1882,96 +1934,129 @@ export default {
               alerts.success('Commit relayed, waiting for confirmation...');
               neo.monitorTransactionConfirmation(res.tx, true)
                 .then(() => {
-                  resolve(res.tx);
-                  this.fetchCommitState(wallets.getCurrentWallet().address);
+                  setTimeout(async () => {
```

Comment:
What's the purpose of the setTimeout here?

Reply:
Ensuring the commit state will be consistent before retrieving it.